lib/terraform/command_switches.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Copyright 2016 New Context Services, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shared command line switches for Terraform
+module CommandSwitches
+  def color_switch
```

Comment:
There should be shared examples defined in `spec/support/terraform/command_switches_examples.rb` that can be included by `apply_command_spec.rb` `plan_command_spec.rb` so they don't have to each implement a test for this.